website/src/data/components/choropleth/props.js

```diff
@@ -65,6 +65,13 @@ const props = [
         controlType: 'quantizeColors',
         group: 'Style',
     },
+    {
```

Comment:
Can you move this to next to other `Base` props please?

Reply:
You got it! Should we add a description since the other base props all have a description?

Reply:
Yes! it could help users understand how does this value affect the chart

Reply:
Ok moved it next to the other base props and added a description.